### ISSUE TITLE
npm release is out of date

### ISSUE THREAD

**Issue by avaragado** [machineboy2045/angular-selectize#117]
https://www.npmjs.com/package/angular-selectize2 reports version 1.2.3.

Please release an updated version for npm.

**Comment by cudasteve** [comment]
It looks like you tried to update package.json to include the 3.0.1 version number but the v3.0.1 tag is below that commit.  So when npm looks up the v3.0.1 tag, it gets the v3.0.1 code (I think) but the package.json file that gets downloaded still says v.1.2.3, which can lead to issues when using npm or other tools like npm-shrinkwrap cuz they'll get tripped up on the version number in that file.  Perhaps just moving the git v3.0.1 tag a few commit forward would help.  Or publishing a v3.0.2 with everything in sync?

**Comment by chrisparton1991** [comment]
Agreed, I've created #151 to add a `main` entry to `package.json`, but it won't help if npm isn't being updated.

**Comment by SteveShaffer** [comment]
I think you should be able to reference the GitHub repo directly in your package dependencies if npm isn't current.

**Comment by adiakritos** [comment]
I wonder if this is the reason that I can't get it to load within my app.js file while using webpack...
